src/rendering/filters/shared/FilterSystem.ts

```diff
@@ -540,7 +540,6 @@ export class FilterSystem implements System
 
     public destroy()
     {
-        // BOOM!
+        // No critical operations to be called in the destroy method.
     }
 }
```

Comment:
If destroy is not needed then we can delete the function and just do `public destroy?: () => void;`

Reply:
Good idea. I applied this to a few other systems with same conditions.